substrate/frame/session/benchmarking/src/inner.rs

```diff
@@ -102,6 +103,71 @@ mod benchmarks {
 		Ok(())
 	}
 
+	#[benchmark]
+	fn rotate_session(v: Linear<1, MAX_VALIDATORS>) -> Result<(), BenchmarkError> {
+		// Create validators 
+		let validators = create_validators::<T>(v, 1000)?;
+
+		// Convert AccountId to ValidatorId 
+		let validator_ids: Vec<T::ValidatorId> = validators.iter()
+			.filter_map(|v| {
+				let account = T::Lookup::lookup(v.clone()).ok()?;
+				T::ValidatorIdOf::convert(account)
+			})
+			.collect();
+
+		if validator_ids.len() != v as usize {
+			return Err("validator conversion failed".into());
+		}
+
+		// Setup: Directly populate Session storage
+		let session_index = 1u32;
+		CurrentIndex::<T>::put(session_index);
+		Validators::<T>::put(&validator_ids);
+
+		// Setup queued keys 
+		let base_keys = T::Keys::decode(&mut TrailingZeroInput::zeroes())
+			.map_err(|_| "decode failed")?;
+
+		// Set NextKeys for all validators to avoid warnings from session handlers
+		for id in validator_ids.iter() {
+			NextKeys::<T>::insert(id, &base_keys);
+		}
+
+		QueuedKeys::<T>::put(
+			validator_ids.iter()
+				.map(|id| (id.clone(), base_keys.clone()))
+				.collect::<Vec<_>>()
+		);
+
+		// Worst-case scenario setup:
+		// 1. Validator set changed (triggers DisabledValidators::kill())
+		QueuedChanged::<T>::put(true);
+
+		// 2. Maximum validators disabled based on DisablingStrategy limit
+		// With default DISABLING_LIMIT_FACTOR=3, max is (v-1)/3 validators
+		// With DISABLING_LIMIT_FACTOR=2, max is (v-1)/2 validators
+		// We use factor=2 for worst case (up to 50% of validators disabled)
+		let disabled_count = v.saturating_sub(1).saturating_div(2);
+		for i in 0..disabled_count {
+			DisabledValidators::<T>::mutate(|d| {
+				d.push((i, OffenceSeverity::default()));
```

Comment:
You can replace this with Default and thus avoid the extra crate import
```suggestion
				d.push((i, Default::default()));
```